node/resolvers/Routes/index.ts

```diff
@@ -38,7 +38,8 @@ const getUserAndPermissions = async (ctx: Context) => {
   const {
     data: { checkUserPermission },
   }: any = await storefrontPermissions
-    .checkUserPermission()
+    // I don't know why this name sender is necessary, maybe it is some policy or permission.
+    .checkUserPermission('vtex.b2b-orders-history@0.x')
```

Comment:
This actually seems like a copy/paste problem. The sender is supposed to be the app sending the request, and it's used for 2 things:

- Metrics - tracking which apps makes requests to which other apps
- App access - making sure callers can only call apps that they directly depend on in their `manifest.json` file.

So to be correct this should always be `vtex.b2b-organizations-graphql` here, as you had before. If this causes any permission problem it's because the manifest.json for this repo is missing store-permissions, which shouldn't be the case since there are other callers passing the sender correctly.

Reply:
Got it! Seems like there are other checks then, probably due to the token also be associated with an app. Let's keep orders-history here in this case, though it's weird.